hack/local-development/local-garden/kubeconfigs/.gitignore

```diff
@@ -1 +0,0 @@
-gardenlet-*.conf
```

Comment:
Can you explain why this .gitignore is removed?

Reply:
When `make start-gardenlet` is ran then we no longer persist the kubeconfig in this folder. Instead we store it in a temporary directory with a client certificate issued via a CSR (https://github.com/gardener/gardener/blob/master/hack/local-development/start-gardenlet#L26-L62). Hence, ignoring these files is no longer needed.